docs/readme.md

```diff
@@ -82,15 +82,38 @@ log.setLevel(flags.logLevel);
 launchChromeAndRunLighthouse('https://example.com', flags).then(...);
 ```
 
+## Configuration
+In order to configure Lighthouse programmatically, you need to pass the config object as the 3rd argument.
+
+**Example:**
+```js
+{
+  extends: 'lighthouse:default',
+  settings: {
+    onlyAudits: [
+      'first-meaningful-paint',
+      'speed-index-metric',
+      'estimated-input-latency',
+      'first-interactive',
+      'consistently-interactive',
+    ],
+  },
+}
```

Comment:
I can see both the benefits and the downsides to having this repeated here and in `configuration.md`.  Seems ok in exchange for the explanation in place? Just more places to update when updates are needed

Reply:
@brendankenny 

> Seems ok in exchange for the explanation in place? 

I'm not too sure what you meant here?

Reply:
> I'm not too sure what you meant here?

I meant, it is duplicating content, which can get lost and fall behind as we update docs in the future and don't hit all the places that need to be updated.

But on the other hand, it means someone reading this doesn't have to click through to another page just to get a simple example of the config, which is kind of nice

Reply:
@brendankenny Yep, I understand the pros and cons, but what is our preference in this case?

Reply:
> what is our preference in this case?

that's why I said

> Seems ok

:)